app/controllers/planning_applications/validation/reporting_types_controller.rb

```diff
@@ -30,7 +30,7 @@ def update
       private
 
       def reporting_type_params
-        params[:planning_application] ? params.require(:planning_application).permit(:reporting_type) : params.permit(:reporting_type)
+        params[:planning_application] ? params.require(:planning_application).permit(:reporting_type, :regulation_3, :regulation_4) : params.permit(:reporting_type, :regulation_3, :regulation_4)
```

Comment:
This might be more readable as an if statement.

```suggestion
        if params[:planning_application]
          params.require(:planning_application).permit(:reporting_type, :regulation_3, :regulation_4)
        else
          params.permit(:reporting_type, :regulation_3, :regulation_4)
        end
```

Or actually, since only the beginning of the chain changes:

```suggestion
        (params[:planning_application] ? params.require(:planning_application) : params).permit(:reporting_type, :regulation_3, :regulation_4)
```